blockstore/apps/api/v1/views/bundles.py

```diff
@@ -0,0 +1,52 @@
+"""
+Views for Bundles and BundleVersions.
+"""
+
+from rest_framework import viewsets
+from rest_framework.generics import get_object_or_404
+
+from blockstore.apps.bundles.models import Bundle, BundleVersion
+
+from ...constants import UUID4_REGEX, VERSION_NUM_REGEX
+from ..serializers.bundles import BundleSerializer, BundleVersionSerializer
+
+
+class BundleViewSet(viewsets.ModelViewSet):
+    """
+    ViewSet for Bundle model.
+    """
+
+    lookup_field = 'uuid'
+    lookup_url_kwarg = 'bundle_uuid'
```

Comment:
Can we also filter `Bundle`s associated with a `Collection`?

Ok to defer this to a subsequent PR, since this is just a first pass at the REST API.

Reply:
Yes, we should have all the filters that make sense. I looked at this briefly but there are [different options](http://www.django-rest-framework.org/api-guide/filtering/) for adding filters plus (like in this case) we will also want to add related resources fields to at least `CollectionSerializer` and `BundleSerializer`. Its a reasonably sized chunk of work so have added it to LX-53.

Reply:
👍